components/Home/InstallTabs/LinuxPanel/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import LocalizedLink from '../../../LocalizedLink';
+import { ShellBox } from '../../../Common';
+import styles from '../index.module.scss';
+import { getNvmLatestVersion } from '../../../../util/getNvmData';
+import type { FC } from 'react';
+
+type Props = {
+  nvmVersion: string;
+};
+
+export const PureLinuxPanel: FC<Props> = ({ nvmVersion }) => {
+  const nvmInstallScriptUrl = `https://raw.githubusercontent.com/nvm-sh/nvm/${nvmVersion}/install.sh`;
+
+  return (
+    <div>
+      <ShellBox textToCopy={`curl -o- ${nvmInstallScriptUrl} | bash`}>
+        <span className={styles.installTextNoSelect}>$</span>
+        <span className={styles.installTextCommand}>curl -o- </span>
+        {nvmInstallScriptUrl}
+        <span className={styles.installTextCommand}> | bash</span>
+      </ShellBox>
+      <ShellBox textToCopy="nvm install --lts">
+        <span className={styles.installTextNoSelect}>$</span>
+        <span className={styles.installTextCommand}>nvm</span> install --lts
+      </ShellBox>
+      {/* @TODO: Verify this link when we implement content */}
+      <LocalizedLink
+        className={styles.installDocsButton}
+        href="/download/package-manager/#nvm"
+      >
+        <FormattedMessage id="components.home.installTabs.readDocs" />
+      </LocalizedLink>
+    </div>
+  );
+};
+
+const LinuxPanel: FC = () => {
+  const [nvmVersion, setNvmVersion] = useState('');
+
+  useEffect(() => {
+    getNvmLatestVersion().then(version => setNvmVersion(version));
```

Comment:
I would recommend if we use the TanStack Query library for handling all async code, it does error handling really well and prevents us from pitfalls of `useEffect`.

Reply:
This lib in't on codebase. So I will take a look to SWR

Reply:
what do you thinks about this code:
```js
 const [nvmVersion, setNvmVersion] = useState<string | null>(null);
  const { data: nvmLatestVersion } = useSWR('nvm-latest-version', () =>
    getNvmLatestVersion()
  );

  useEffect(() => {
    if (nvmLatestVersion) {
      setNvmVersion(nvmLatestVersion);
    }
  ```
  personally I don't found that better